frontend/src/lib/load_spells.js

```diff
@@ -0,0 +1,15 @@
+import { getAll, getBySlug } from "./load_data";
+
+const getSpellsBySlug = async (slug) => {
+  return await getBySlug('spells', slug);
+};
+
+const getSpells = async (query) => {
+  const { searchQuery } = query;
+  if (searchQuery) {
+    query['searchFilter'] = `filter[name_cont_any]=${searchQuery}`;
```

Comment:
I think it would be good to change the filter so that it also filters by incantation, as this is often more known, e.g. _Accio_ for the Summoning Charm or _Wingardium Leviosa_ for the Levitation Charm. 
Changing it to `filter[name_or_incantation_cont_any]` should probably do it, maybe you can test this, again.

Reply:
This was a nice and easy solution 👍🏽